client/src/store/new-request-module.ts

```diff
@@ -376,6 +403,12 @@ export class NewRequestModule extends VuexModule {
         'Reported as Corporate tax',
         'Has name protection in BC'
       ],
+      mveBlurbs: [
+        'A type of corporations which has a benefit to  the community.  They are intended to bridge the gap between ' +
```

Comment:
"corporations" should be singular?

Also, should be "xxx, which yyy" or "xxx that yyy", if possible. (Here, I prefer the "that" form.)

Second sentence refers to "they", which is plural. Can you use "it" to solve this?

Reply:
lol for sure :)